### PR TITLE
Added JSON Lint link to check for linting error.

### PR DESCRIPTION
**Pull request by MAYANKpandey14**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarised content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot 2023-01-28 at 1 12 44 AM

## Note to reviewers
This pull request is regarding this issue:-
Add a link to jsonlint in the quickstart guide #3914

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4135"></a>

### CODE REVIEW

**Review by ChinmayMhatre — DISMISSED** [review]
Looks good to me! Was wondering if "check for syntax error" would be better and "error free"

File: pages/docs/quickstart.mdx [hunk 1]
```diff
@@ -108,6 +108,8 @@ To add more links add another object inside the links collection. For example:
 10. Scroll to the bottom and commit new file by adding the commit message that starts `data: ` followed by your GitHub username (for example `data: SaraJaoude`)
 
 You can skip `optional extended message`
+**NOTE:-**
+Please use [JSON Lint](https://jsonlint.com) to check if your JSON file is `error free` before commiting your pull request.
```
Comment (eddiejaoude) [inline pages/docs/quickstart.mdx:112]: We use this "style" of notes in the docs...

```suggestion
**NOTE**: Please use the online [JSON Lint](https://jsonlint.com) tool to check if your JSON file is *error free* before committing your pull request.
```

File: pages/docs/quickstart.mdx [hunk 2]
```diff
@@ -108,6 +108,8 @@ To add more links add another object inside the links collection. For example:
 10. Scroll to the bottom and commit new file by adding the commit message that starts `data: ` followed by your GitHub username (for example `data: SaraJaoude`)
 
 You can skip `optional extended message`
+**NOTE:-**
```
Comment (eddiejaoude) [inline pages/docs/quickstart.mdx:111]:
```suggestion

```

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Looks good 👍 Made suggestion to be consistent with our docs

**Comment by MAYANKpandey14** [comment]
> Looks good 👍 Made suggestion to be consistent with our docs

Yeah sure Eddie, I'll make the changes as suggested.

**Review by eddiejaoude — APPROVED** [review]
Thanks 👍

**Comment by eddiejaoude** [comment]
@ChinmayMhatre made a good suggestion 👍 this can come in another PR

**Comment by MAYANKpandey14** [comment]
Thank you guys @eddiejaoude @ChinmayMhatre  for helping me as am new to Open source .If any more changes are to be done feel free to ping me here .

**Pull request merged**